shared/localization/locales/lv.json

```diff
@@ -1758,7 +1764,7 @@
     "message": "Šīs pārbaudes ir vajadzīgas saskaņā ar standarta [PWA kontrolsarakstu](https://developers.google.com/web/progressive-web-apps/checklist), taču Lighthouse neveic tās automātiski. Tās neietekmē jūsu rezultātu, taču ir svarīgi pārbaudīt šos lietotnes aspektus manuāli."
   },
   "lighthouse-core/config/default-config.js | pwaCategoryTitle": {
-    "message": "PWA"
+    "message": "PTL"
```

Comment:
Is it bad practice to say "leave the whole thing untranslated" in a translator comment?

Reply:
we should just not have this be a i18n string

Reply:
I'd be ok to override this once and make the changes after release